build/Targets/Settings.props

```diff
@@ -186,7 +188,7 @@
       <PropertyGroup>
         <ProjectLanguage>C++</ProjectLanguage>
```

Comment:
It looks like we have a single project `CLibraryShim` that doesn't actually import our common props/targets (by design since it is a test project).

It only imports:
```
	Line 29:   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
	Line 58:   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
	Line 64:     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
	Line 67:     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
	Line 70:     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
	Line 73:     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
	Line 148:   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
```